pages/colonydetail/[id].js

```diff
@@ -1,19 +1,21 @@
 import { useRouter } from "next/router";
-import ColonyCard from "@/components/ColonyCard";
+import Card from "@/components/Card";
 import ColonyQuestions from "@/components/ColonyQuestions";
-import { StyledLink } from "../locationdetail/[id]";
+import { StyledBackLink } from "@/components/StyledLinks";
 
 export default function ColonyDetailsPage({ colonies }) {
   const router = useRouter();
   const id = router.query.id;
 
-  const result = colonies.find((colony) => colony.id === id);
+  const currentColony = colonies.find((colony) => colony.id === id);
+
+  console.log("currentColony", currentColony);
 
   return (
     <>
-      <ColonyCard colony={result} />
+      <Card text={currentColony?.colonyName} />
       <ColonyQuestions />
-      <StyledLink href={`/locationdetail/${id}`}>←</StyledLink>
+      <StyledBackLink href={`/locationdetail/${currentColony.id}`} />
```

Comment:
The currentColony knows his `locationId`. So for the href you must use `currentColony.locationId`. That is where the location of the current colony is stored.
Please try it out and give me a response.

Reply:
That is so weird, actually I had the link exactly like that and it worked fine, it was just that the vercel deployment failed. Now it works. Thanks! 

And regarding the list behind the navbar: For me, it worked fine (localhost as well as vercel preview), but I gave the main a margin to the bottom as you suggested Hopefully it works for you as well now.